src/app/models/trip.js

```diff
@@ -0,0 +1,65 @@
+import Backbone from 'backbone';
+
+const Trip = Backbone.Model.extend({
+  urlRoot: 'https://ada-backtrek-api.herokuapp.com/trips',
+  // const CONTINENTS = ['', ];
+
+
+  validate(attributes) {
+    // Note the argument. We will read attribute values from
+    // here instead of calling this.get()
+
+    // Format of errors: same as Rails!
+    // {
+    //   title: ['cannot be blank', 'already taken'],
+    //   author: ['cannot be blank']
+    // }
+    // continents mabye client-side validation??
+    const CONTINENTS = ['Africa', 'Antartica', 'Asia', 'Australasia', 'Europe', 'North America', 'South America']
+
+    const errors = {};
+    if (!attributes.name) {
+      errors.name = ['cannot be blank'];
+    }
+
+    if (!attributes.continent) {
+      errors.continent = ['cannot be blank'];
+    }
+
+    if (!CONTINENTS.includes(attributes.continent)) {
```

Comment:
Nice, but you might want to check to see if `errors.continent ` is defined and if it is pushing the error message into the array.